monitors.go

```diff
@@ -69,7 +68,7 @@ func (m *Monitor) JobUpdate(updateKey aurora.JobUpdateKey, interval int, timeout
 	}
 
 	fmt.Println("Timed out")
```

Comment:
Return for JobUpdate is tuple: (bool, error). If we choose to throw error for timeouts then only error is sufficient. 
Same goes for Instances 
Instances(key *aurora.JobKey, instances int32, interval int, timeout int) (bool, error) ..
what is your preference?